main/vmWatch.go

```diff
@@ -36,7 +37,10 @@ const (
 
 const (
 	AllowVMWatchCgroupAssignmentFailureVariableName string = "ALLOW_VMWATCH_CGROUP_ASSIGNMENT_FAILURE"
-	RunningInDevContainerVariableName string = "RUNNING_IN_DEV_CONTAINER"
+	RunningInDevContainerVariableName               string = "RUNNING_IN_DEV_CONTAINER"
+	AppHealthExecutionEnvironmentProd                      = "Prod"
+	AppHealthExecutionEnvironmentTest                      = "Test"
+	AppHealthPublisherNameTest                             = "Microsoft.ManagedServices.Edp"
```

Comment:
nit: add type string to the rest
```suggestion
	AllowVMWatchCgroupAssignmentFailureVariableName string = "ALLOW_VMWATCH_CGROUP_ASSIGNMENT_FAILURE"
	RunningInDevContainerVariableName               string = "RUNNING_IN_DEV_CONTAINER"
	AppHealthExecutionEnvironmentProd                      = "Prod"
	AppHealthExecutionEnvironmentTest                      = "Test"
	AppHealthPublisherNameTest                             = "Microsoft.ManagedServices.Edp"
```